addons/maintenance/views/maintenance_views.xml

```diff
@@ -728,6 +744,24 @@
         </field>
     </record>
 
+    <record id="maintenance_team_view_kanban" model="ir.ui.view">
```

Comment:
@kic-odoo : same question.. the kanvan view for maintenance.team already exists.

Reply:
@kic-odoo : Ok, I understood, maybe add 
`<field name="view_id" ref="maintenance_team_view_kanban"/>`
in maintenance_team_action_settings

Reply:
@kic-odoo : What about this?

Reply:
@adr-odoo i have added view_ids in maintenance_team_action_settings.another kanban view is dashboard.

Reply:
My bad.

Reply:
its ok.